app-network/mitmproxy/spec

```diff
@@ -1,5 +1,12 @@
-VER=7.0.4
-SRCS="https://github.com/mitmproxy/mitmproxy/archive/v$VER.tar.gz"
-CHKSUMS="sha256::8728d18c69053f0043acebcdabf46f2eeea51f0f0b60c528e1d356cf48ed2ca2"
+MITMPROXY_VER=11.0.0
```

Comment:
You should define this as UPSTREAM_VER so that `aosc-findupdate` can find updates.